build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -199,6 +205,24 @@ new file mode 100644
 +        android:title="@string/options_adblock_edit_label"
 +        android:fragment="org.chromium.chrome.browser.settings.AdBlockEditor" />
 +
++    <org.chromium.components.browser_ui.settings.SpinnerPreference
++        android:key="adblock_period_spinner"
```

Comment:
Can you please change all the occurrencies of this to `frequency`?

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/b11ab2588e787d6349664120c0ccdb7827cab201